### ISSUE TITLE
Path separator recognized as binary operator in Markdown file

### ISSUE THREAD

**Issue by schillic** [julia-vscode/StaticLint.jl#364]
The formatter also works on Markdown files (documentation), but it does not recognize path separators correctly (and rather reads them as binary operators and adds spaces around them).

#### Before formatting

```julia
![Test](pics/a.png)
```

#### After formatting

```julia
![Test](pics / a.png)
```

**Comment by schillic** [comment]
Sorry, I think this is the wrong place to report this. I went to https://github.com/domluna/JuliaFormatter.jl/issues/712.
